Assets/Drag.cs

```diff
@@ -10,23 +10,52 @@ public class Drag : MonoBehaviour
     Vector3 startingPosition;
     GameObject clone;
     Boolean triggerFlag;
+    Boolean enableFlag;
+
+    public Console console;
+
+    private MeshRenderer renderer;
+    private MeshRenderer clone_renderer;
 
     private void Awake()
     {
         startingPosition = transform.position;
+        renderer = GetComponent<MeshRenderer>();
         manipulationHandler = GetComponent<ManipulationHandler>();
         manipulationHandler.OnManipulationStarted.AddListener(StartedMotion);
         manipulationHandler.OnManipulationEnded.AddListener(StoppedMotion);
+        enableFlag = true;
     }
 
     public void OnTriggerEnter(Collider other)
     {
         triggerFlag = true;
+        if(clone_renderer != null)
+        {
+            foreach (var mat in clone_renderer.materials)
+            {
+                mat.SetFloat("_Outline", 0.15f);
+            }
+        }
     }
 
     public void OnTriggerExit(Collider other)
     {
         triggerFlag = false;
+        if (clone_renderer != null)
+        {
+            foreach (var mat in clone_renderer.materials)
+            {
+                mat.SetFloat("_Outline", 0f);
+            }
+        }
+        if (renderer != null)
```

Comment:
you might be able to get away with a elvis operator here so:

```C#
foreach(Material mat in render?.materials){
    mat.SetFloat("_Outline",0f);
}
```
and get rid of the if statement, same for the `clone_renderer` loop above